src/python/bot/tasks/minimize_task.py

```diff
@@ -1395,13 +1419,14 @@ def minimize_file(file_path,
 
   # Specialized minimization strategy for javascript.
   if file_path.endswith('.js'):
-    return do_js_minimization(test_function, get_temp_file, data, deadline,
-                              threads, cleanup_interval, delete_temp_files)
+    js_tokenizer = AntlrTokenizer(JavaScriptLexer)
```

Comment:
in this current form the change will unconditionally switch the minimization for JS files to the antlr-based tokenizer, but I guess we don't want it right now. Will need to decide whether we make this behavior depend on some additional parameter, or disable for now. Not an issue you can address right away, but we'll need to think before landing this

Reply:
@mpherman2 do you know what was @mbarbella-chromium 's plan on this? I don't think we should switch to the new minimizer until we're happy with its performance

Reply:
Not sure, he did not say. I think either setting a parameter or just disabling it entirely for now are both fine. What do you think?

Reply:
I would disable it for now. Enabling it is fast and easy.

Reply:
Sounds good